dockerfiles/attestation-service/Dockerfile

```diff
@@ -1,9 +1,6 @@
 FROM circleci/node:10
```

Comment:
The PR is using again the `circle/node:10`? Or it was not merged yet?

Reply:
Good catch, it just needed to be merged. Thanks